src/main/java/org/jitsi/jicofo/recording/jibri/JibriRecorder.java

```diff
@@ -323,33 +340,11 @@ private void processJibriIqFromMeet(JibriIq iq, XmppChatMember sender)
             if (jibriJid == null)
             {
                 sendErrorResponse(
-                    iq, XMPPError.Condition.service_unavailable, null);
+                        iq, XMPPError.Condition.service_unavailable, null);
                 return;
             }
 
-            final String roomName = MucUtil.extractName(senderMucJid);
-
-            try
-            {
-                final XMPPError err = startJibri(
-                        jibriJid, roomName, iq.getStreamId());
-                if (err == null)
-                {
-                    // ACK the original request
-                    sendResultResponse(iq);
-                }
-                else
-                {
-                    sendErrorResponse(
-                            iq, XMPPError.Condition.interna_server_error, null);
-                }
-            }
-            catch (final SmackException.NoResponseException e)
-            {
-                sendErrorResponse(
-                        iq, XMPPError.Condition.request_timeout, null);
-                return;
-            }
+            scheduleRetryStartJibri(jibriJid, iq);
```

Comment:
The whole else if should be changed to get rid of returns which are error prone(that is not related to the changes in the PR itself, but was like that from the start)